server/methods.js

```diff
@@ -1,14 +1,13 @@
-if(Meteor.isServer) {
```

Comment:
This isn't needed because only the server will run things inside the server folder